packages/block-editor/src/components/block-manager/index.js

```diff
@@ -21,11 +25,13 @@ import BlockManagerCategory from './category';
  * @param {Array}    props.blockTypes         An array of blocks.
  * @param {Array}    props.selectedBlockTypes An array of selected blocks.
  * @param {Function} props.onChange           Function to be called when the selected blocks change.
+ * @param {boolean}  props.showSelectAll      Whether to show the select all checkbox.
  */
 export default function BlockManager( {
 	blockTypes,
 	selectedBlockTypes,
 	onChange,
+	showSelectAll = true,
```

Comment:
This component is private, so we don't need to care about backward compatibility.